src/box/box.cc

```diff
@@ -1529,6 +1529,18 @@ box_check_replication_anon(void)
 	return anon;
 }
 
+static double
+box_check_replication_anon_gc_timeout(void)
```

Comment:
Why did we decide to go with `replication_anon_gc_timeout` and not `wal_anon_gc_timeout`. It seems it has little to do with replication but it's definitely related to gc. And we already have `wal_retention_period`

Reply:
> We decided to implement garbage collector for anonymous replicas, not WAL GC consumers. The name was changed correspondingly.

The name should not depend on internal implementation. `wal_retention_period` is a number of seconds to save xlogs after last use. `replication_anon_gc_timeout` does the same thing: number of seconds to save xlogs for anon replica after its disconnection. From the user point of view, it's non-obvious, why there're two almost the same options, which are named very differently.

Reply:
No, `replication_anon_gc_timeout` - timeout after which we delete an anonymous replica.